src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs

```diff
@@ -1553,7 +1553,7 @@ public void When_Ignore_Surrounding_Whitespace()
 				<LineBreak />
 				   You can construct URLs and access their parts. For URLs that represent local files, you can also manipulate properties of those files directly.
 				<LineBreak />
-				   AfterLineBreak 
+				   AfterLineBreak
```

Comment:
I had whitespace changes turned off in GitHub and didn't see this change. This was causing the test failure in this PR. This is caused by https://github.com/unoplatform/uno/blob/8a0bdce947b3e15caf97ed3adf47a087508a9506/.editorconfig#LL5C1-L5C26 which is very bad. We should never set this option to true.